Document IsPublishable (#24134)
dotnet/docs: docs/core/tools/dotnet-publish.md

### MSBuild

> The `dotnet publish` command calls MSBuild, which invokes the `Publish` target. If the `IsPublishable` property in a project's *.csproj* file is set to `false`, the `Publish` target can't be invoked, and in that case the `dotnet publish` command doesn't do anything with the project.

Reviewer: > doesn't do anything with the project

Does it raise an error in such a case?

Author: No error because this is expected behavior. If you have a solution with two projects, one of which is not publishable, when you run `dotnet publish` on the solution you get a successful run that publishes only the publishable project.

> The `dotnet publish` command calls MSBuild, which invokes the `Publish` target. If the `IsPublishable` property in a project's *.csproj* file is set to `false`, the `Publish` target can't be invoked, and in that case the `dotnet publish` command doesn't do anything with the project.

Reviewer 2: ```suggestion
The `dotnet publish` command calls MSBuild, which invokes the `Publish` target. If the [`IsPublishable` property](../project-sdk/msbuild-props.md#ispublishable) is set to `false` for a particular project, the `Publish` target can't be invoked, and the `dotnet publish` command doesn't do anything with the project.
```

> The `dotnet publish` command calls MSBuild, which invokes the `Publish` target. If the [`IsPublishable` property](../project-sdk/msbuild-props.md#ispublishable) is set to `false` for a particular project, the `Publish` target can't be invoked, and the `dotnet publish` command doesn't do anything with the project.

Reviewer 2: ```suggestion
The `dotnet publish` command calls MSBuild, which invokes the `Publish` target. If the [`IsPublishable` property](../project-sdk/msbuild-props.md#ispublishable) is set to `false` for a particular project, the `Publish` target can't be invoked, and the `dotnet publish` command ignores the project.
```
